elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/OidcConfigService.java

```diff
@@ -255,6 +256,10 @@ private String getJSON(ModelNode deployment, boolean convertToRealmConfiguration
                 json.get(REALM).set(authServerUrlAndRealm[1]);
                 json.remove(PROVIDER_URL);
             }
+            String scope = json.get(SCOPE).asStringOrNull();
```

Comment:
This block of code (the `convertToRealmConfiguration` block) is related to `secure-server` configuration specifically so we don't need to add anything here.

Notice this block, https://github.com/wildfly/wildfly/blob/main/elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/OidcConfigService.java#L239, where the JSON values are set using `SecureDeploymentDefinition.ALL_ATTRIBUTES` so once the SCOPE definition has been added to the `SecureDeploymentDefinition` class as mentioned above and once `ALL_ATTRIBUTES` has been updated to include that, this will already get set automatically.